media/js/base/send-to-device.js

```diff
@@ -49,34 +50,48 @@ if (typeof Mozilla === 'undefined') {
     // static value for user country code
     SendToDevice.COUNTRY_CODE = '';
 
-    SendToDevice.prototype.geoCallback;
+    SendToDevice.prototype.geoCallback; // jshint ignore:line
 
     /**
      * Initialise the form messaging and bind events.
      */
     SendToDevice.prototype.init = function() {
         if (this.$widget.length === 1) {
-            this.checkLocation();
+            this.getLocation();
             this.bindEvents();
         }
     };
 
     /**
-     * Checks to see if the visitor is located in the US
-     * using MLS https://location.services.mozilla.com/api
+     * Gets the country code of the location of the user
+     * using bedrock's /country-code.json service
      */
-    SendToDevice.prototype.checkLocation = function() {
+    SendToDevice.prototype.getLocation = function() {
         var self = this;
-        var key = this.$widget.data('key');
 
-        // should location.services.mozilla.com be slow to load,
+        // if a dev has added ?geo=<country code> to the URL
+        // we can skip the geo lookup and act as if it worked
+        if (window.location.href.indexOf('?geo=') !== -1) {
+            var urlRe = /\?geo=([a-z]{2})/i;
+            var match = urlRe.exec(window.location.href);
```

Comment:
Could maybe test against `window.location.search` here (and above) instead of the full href?